PolishWithBlondes/src/components/hero/hero.scss

```diff
@@ -0,0 +1,51 @@
+.hero {
+  background-color: var(--color-light-bg);
+  color: var(--primary-font-color);
+  font-family: var(--font-family);
+  font-size: 14px;
+  padding: 40px;
+  display: flex;
+  max-width: 1060px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.col {
+  flex: 1;
+}
+.left {
+  padding-right: 50px;
```

Comment:
It's not the best practice to use padding as a spacer between columns.
Try setting the width of both columns (maybe percentage, like 48% and 48%) and then use justify-content: space-between

Reply:
For the desktop breakpoint and upwards I set separate widths for the two columns in percentages, because I want the image to take up slightly more space. I also added flex-shrink: 0 for the image